Kickstarter-iOS/Views/Controllers/ProjectPamphletViewController.swift

```diff
@@ -24,12 +24,12 @@ public final class ProjectPamphletViewController: UIViewController {
 
   @IBOutlet private var navBarTopConstraint: NSLayoutConstraint!
 
-  private let backThisProjectContainerViewMargins = Styles.grid(3)
-  private let backThisProjectContainerView: UIView = {
-    UIView(frame: .zero) |> \.translatesAutoresizingMaskIntoConstraints .~ false
+  private let pledgeCTAContainerViewMargins = Styles.grid(3)
+  private let pledgeCTAContainerView: PledgeCTAContainerView = {
+    PledgeCTAContainerView(frame: .zero) |> \.translatesAutoresizingMaskIntoConstraints .~ false
   }()
 
-  private let backThisProjectButton: UIButton = {
+  private let pledgeCTAButton: UIButton = {
```

Comment:
Hey @cdolm92 I know this is already merged but did we forget to remove this button? I don't see it being used in this file. Same goes for the `backThisProjectTapped` function in this file.

Reply:
Thanks for pointing this out! I will make a card to remove this.